docs/fidesops/docs/guides/saas_postprocessors.md

```diff
@@ -28,6 +28,32 @@ endpoints:
 
 Note: Order matters as it's defined in the config. In the above example, unwrap will be run first, then the output of unwrap will be used in the filter strategy.
 
+### For Update/Delete/GDPR endpoints
+
+For postprocessors defined in update requests, we're postprocessing the access request rows that were retrieved in a format that's needed for update statement.
+
+For example, if we need to return the following in an access request:
+
+```json
+{"recipient": "test@email.com",
+  "subscriptions": [{
+    "id": "123",
+    "subscribed": "TRUE"
+  }]}
+```
+
+And we needed to perform an update request for each item within `subscriptions`, where `subscribed` = `TRUE`, then we'd need the following config for our update request:
```

Comment:
"following config for our post-processor"?

Reply:
this bit: `data_path: subscriptionStatuses` is external to post-processors, so I wasn't sure if I should write up a separate bit in `sass_config.md`, or just address this use case more generally in the update request config as I did here.